<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/infant/dkan_outdoor/2019-01-13-15-46-58.bag -s 60"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_transform_cuda)/rviz_config/velodyne.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- main -->
	<node pkg="pc_transform_cuda" type="pc_transform_pcl" name="pc_transform_pcl" output="screen">
		<remap from="/point_cloud" to="/velodyne_points"/>
		<remap from="/point_cloud/transformed" to="/velodyne_points/transformed"/>
		<param name="x_m" type="double" value="1.0"/>
		<param name="y_m" type="double" value="2.0"/>
		<param name="z_m" type="double" value="3.0"/>
		<param name="r_deg" type="double" value="10.0"/>
		<param name="p_deg" type="double" value="20.0"/>
		<param name="y_deg" type="double" value="30.0"/>
	</node>
</launch>
